python-lib/tagger.py

```diff
@@ -1,38 +1,45 @@
-"""Module to prepare the tagging of the documents
-splits on sentences and instanciates a Matcher to tag the documents"""
+"""Main module to tag the documents"""
```

Comment:
Nitpick: since "tagging" is a generic need, which may be necessary for future recipes in this plugin, could be better to rename this file to "ontology_tagger"